java/src/jmri/jmrix/tams/TamsSensorManager.java

```diff
@@ -18,23 +18,23 @@
  *
  * @author Kevin Dickerson Copyright (C) 2009
  * @author Jan Boen and Sergiu Costan
- * 
+ *
```

Comment:
This is another case of why did this file get included in the PR?  I'm also questioning why there were spaces inserted in the middle of three asterixs in this file.. in log.debug statements that are commented out.

Reply:
My guess is that I opened it (in BBEdit which auto-magicly fixes tabs, trailing white-spaces, etc.) while "borrowing" (read "plagiarizing") code and I didn't notice it and it got checked in. Sorry.

Reply:
fair enough.

The thing I really didn't like was it changed *** to * ** inside of a string which was in a line of  code that was commented out.  

That is just wrong on so many levels.

Reply:
Reverting TamsSensorManager.java